DiscordChatExporter.Core/Exporting/Writers/Html/PreambleTemplate.cshtml

```diff
@@ -254,6 +254,22 @@
             unicode-bidi: bidi-override;
         }
 
+        .chatlog__system-message{
+            color: #96989D
+        }
+
+        .chatlog__system-message-reference-link{
+            font-weight: 500;
+            color: #ffffff            
+        }
+
+        .chatlog__system-message-icon{
+            width: 16px;
+            height: 16px;
+            margin-right: 0.25rem;
+            border-radius: 50%;            
```

Comment:
```suggestion
            color: #ffffff
        }

        .chatlog__system-message-icon{
            width: 16px;
            height: 16px;
            margin-right: 0.25rem;
            border-radius: 50%;
```

Reply:
Thanks for pointing it out. Removed the whitespaces^^